src/main/java/uk/gov/hmcts/ccd/CommsConfiguration.java

```diff
@@ -1,15 +0,0 @@
-package uk.gov.hmcts.ccd;
-
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.client.RestTemplate;
-
-@Configuration
-public class CommsConfiguration {
```

Comment:
This is a cleanup - it was never used turns out. There is another RestTemplateConfiguration class that build the comms config.